README.md

```diff
@@ -113,7 +113,47 @@ given in the section [Running using Python](#Running-with-Python).
 
 ### Update the codebase when a new OpenAPI spec is available
 
-TBA
+When a new API has been released there are 2 ways to update this repository
+with the new specification.
+
+The procedure in both cases starts by checking out the [nlp-sandbox-schemas](https://github.com/Sage-Bionetworks/nlp-sandbox-schemas) repository that contains the
+newest API Specification of all NLP Sandbox projects. Make sure you have the prerequisits installed for that project
+before proceeding. These instructions assume you have checked out ALL projects to your home directory
```

Comment:
> These instructions assume you have checked out ALL projects to your home directory

Why request the developer of a Date Annotator to checkout all our repositories?

Reply:
"Projects" is a loose term.

More generally, the developer of an NLP Tool should rely on the latest official specification published and not requiring them to build the edge version. The instructions should start by 1) going to https://github.com/Sage-Bionetworks/nlp-sandbox-schemas, 2) identify if a newer version of the spec has been released since the developer created his tool OR since last time he updated it, 3) Download the latest version of the spec: https://sage-bionetworks.github.io/nlp-sandbox-schemas/date-annotator/latest/openapi.yaml, 4) npx @openapitools/openapi-generator-cli generate -i openapi.yaml -g python-flask -o server.

Note that I use `npx` which is more user friendly for the developer as it does not requires him to install openapi-generator-cli before running the command.

Reply:
done